Assets/HoloToolkit-Preview/QRTracker/Scripts/SpatialGraphCoordinateSystem.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+#if WINDOWS_UWP
+using Windows.Perception.Spatial;
+#endif
+namespace HoloToolkit.Unity
+{
+    public class SpatialGraphCoordinateSystem : MonoBehaviour
+    {
+#if WINDOWS_UWP
```

Comment:
This class behaves so different for UWP and non-UWP. Is there a more clever way to do this without pragmas everywhere?

Reply:
This class is do-nothing for non UWP. Let me know if we have a better way to achieve that..